src/index.html

```diff
@@ -5,9 +5,273 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="./images/favicon.png">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;1,400&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <div class="header__top">
+        <div class="header__logo logo"></div>
+        <div class="header__phone">+8 800 555 35 35</div>
+        <div class="header__order">
+          <a href="tel:+88005553535" class="header__link">Call To Order</a>
+        </div>
+      </div>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--active">Home</li>
+          <li class="nav__item">
+            <a href="#about-us" class="nav__link">About Us</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gallery" class="nav__link">Gallery</a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">Consultation</a>
+          </li>
+          <li class="nav__item">
+            <a href="#testimonials" class="nav__link">Testimonials</a>
+          </li>
+          <li class="nav__item">
+            <a href="#contact" class="nav__link">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="section" id="about-us">
+      <div class="info">
+        <h1 class="info__title">Miami <strong>best</strong> Real estate
+          <a
+            target="blank"
+            href="https://www.google.com/maps/place/400+N+1st+Ave+%23700,+Minneapolis,+MN+55401,+%D0%A1%D0%A8%D0%90/data=!4m2!3m1!1s0x52b3328fd8912e53:0x4fc8362b3bad3155?sa=X&ved=2ahUKEwj8iq-otuHrAhXjAxAIHXc6CwEQ8gEwAHoECAsQAQ"
+            class="link"
+          >here
+          </a>
+        </h1>
+        <div class="info__text">
+          <p><strong>Over 50 years experience in love.</strong><br> Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
+          <p> The market leader in real estate since 2003.<br> Awards Real Estate Company, 2010, 2011, 2013. <br> More than 200 satisfied customers premium segment. <br> We focus only on new buildings and high-level doskanalno know this area better than other specialists. <br> Working directly with developers - so have more useful information about the profitable deals.</p>
```

Comment:
открывающий тег должен быть на том же месте где и закрывающий
```suggestion
          <p>
            The market leader in real estate since 2003.<br> Awards Real Estate Company, 2010, 2011, 2013. <br> More than 200 satisfied customers premium segment. <br> We focus only on new buildings and high-level doskanalno know this area better than other specialists. <br> Working directly with developers - so have more useful information about the profitable deals.
          </p>
```